src/scrot.c

```diff
@@ -448,33 +448,24 @@ Window scrotGetWindow(Display *display, Window window, int x, int y)
 void scrotGrabMousePointer(const Imlib_Image image, const int xOffset,
     const int yOffset)
 {
-    XFixesCursorImage *xcim = NULL;
-    uint32_t *pixels = NULL;
-    /* Get the X cursor and the information we want. */
-    if ((xcim = XFixesGetCursorImage(disp)) == NULL) {
```

Comment:
Also worthy of note is that `xcim` gets initialized with a value that is never used, that seems to be everywhere in scrot. Searching for `= NULL` in scrot.c reveals many instances.